src/calculator.js

```diff
@@ -36,14 +36,14 @@ exports.multiply = (x, y) => {
   return x * y;
 };
 
-exports.divide = (x, y) => {
+exports.divide = (x,y) => {
```

Comment:
A space is required after &#39;,&#39;.

###### See http://eslint.org/docs/rules/comma-spacing for details about this rule